frontend/ui/networkmgr.lua

```diff
@@ -97,7 +97,7 @@ end
 
 function NetworkMgr:getWifiMenuTable()
     return {
-        text = _("Wifi connection"),
+        text = _("Wi-Fi connection"),
```

Comment:
This one doesn't could go either way. As a menu item the brusqueness of any spelling of *wifi* is probably desirable over something like *wireless connection* or *wireless network connection*.